pkg/api/operators/apis/experiment/v1alpha2/constants.go

```diff
@@ -27,4 +27,7 @@ const (
 
 	// Default value of Spec.TemplatePath
 	DefaultTrialTemplatePath = "defaultTrialTemplate.yaml"
+
+	OptimizationTypeMinimize = "minimize"
```

Comment:
Do we need this? Isn't it already defined in types.go?

Reply:
I will remove it